.github/workflows/spell_check.yml

```diff
@@ -26,5 +26,11 @@ jobs:
 
       - name: Run cspell
         run: |-
-          cspell lint "**"
-          echo -e "\e[91mNote: If the detected words are expected, add them into ".cspell.json" which is at the repo root folder.\e[0m" 
+          cspell lint "**" > cspell_check.log || true
```

Comment:
Do we need to also redirect stderr to the log?

Reply:
After testing, this command somehow outputs detected mis-spell words info into `stdout` and outputs the normal scan info into `stderr`, and we want to print the former info to summary.